Lib-Open5e/o5e_Open5e_get.command

```diff
@@ -0,0 +1,4 @@
+[h: slug = arg (0)]
+[h: url = "https://api.open5e.com/monsters" + slug]
```

Comment:
At some point, we'll be getting more than monsters. Either rename to getMonsters or wrap with getMonsters